nordicsemi/__main__.py

```diff
@@ -187,6 +187,15 @@ def convert(self, value, param, ctx):
 ]
 DEFAULT_BOOT_VALIDATION = 'VALIDATE_GENERATED_CRC'
 
+KEY_CHOICE = ['pk', 'sk']
+KEY_FORMAT =\
```

Comment:
PEP8 discourages use of `\` line continuations. Move the opening bracket up instead.